lib/nimbusRecipe.ts

```diff
@@ -250,4 +252,18 @@ null,
 
     return `https://experimenter.services.mozilla.com/nimbus/${encodeURIComponent(this._rawRecipe.slug)}/summary#${screenshotsAnchorId}`
   }
+
+  /**
+   * @returns true if this recipe is an experiment recipe not in rollout.
+   */
+  isExpRecipe() {
+    return !this._rawRecipe.isRollout
+  }
+
+  /**
+   * @returns true if this recipe is currently a message rollout experiment.
+   */
+  isMsgRolloutRecipe() {
+    return this.usesMessagingFeatures() && this._rawRecipe.isRollout
+  }
```

Comment:
I'd prefer we keep the API simple and composable.  How about just having the callers use `usesMessagingFeatures()` and `!isExpRecipe()` and omitting this function?